va/va_dec_av1.h

```diff
@@ -0,0 +1,546 @@
+             */
+            uint32_t        delta_lf_multi                              : 1;
+
+            /** \brief read_tx_mode
+             *  value range [0..6]
+             */
+            uint32_t        tx_mode                                     : 2;
+
+            /** \brief read_frame_reference_mode
+             */
+            uint32_t        reference_mode                              : 2;
+
+            uint32_t        reduced_tx_set_used                         : 1;
+
+            uint32_t        skip_mode_present                           : 1;
+
+            uint32_t        reserved                                    : 19;
+        } bits;
+        uint32_t            value;
+    } mode_control_fields;
+
+    /** \brief CDEF parameters
+     */
+    /*  value range [0..3]  */
+    uint8_t                 cdef_damping_minus_3;
+    /*  value range [0..3]  */
+    uint8_t                 cdef_bits;
+    /*  value range [0..63]  */
+    uint8_t                 cdef_y_strengths[8];
+    /*  value range [0..63]  */
+    uint8_t                 cdef_uv_strengths[8];
+
+    /** \brief loop restoration parameters
+     */
+    union
+    {
+        struct
+        {
+            uint16_t        yframe_restoration_type                     : 2;
+            uint16_t        cbframe_restoration_type                    : 2;
+            uint16_t        crframe_restoration_type                    : 2;
+            uint16_t        lr_unit_shift                               : 2;
+            uint16_t        lr_uv_shift                                 : 1;
+            uint16_t        reserved                                    : 7;
+        } bits;
+        uint16_t            value;
+    } loop_restoration_fields;
+
+    /** \brief global motion
+     */
+    VAWarpedMotionParamsAV1 wm[7];
+
+    /**@}*/
+
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t                va_reserved[VA_PADDING_MEDIUM];
+} VADecPictureParameterBufferAV1;
+
+
+/**
+ * \brief AV1 Bit Stream Parameter Buffer Structure
+ *
+ * This structure conveys parameters related to bit stream data and should be
+ * sent once per tile.
+ *
+ * Bit stream data buffer of VASliceDataBufferType is used
+ * to send the bitstream.
+ *
+ * Please note that host decoder is responsible to parse out the
+ * per tile information. And the bit stream in sent to driver in per
+ * tile granularity.
+ */
+typedef struct _VABitStreamParameterBufferAV1
+{
+    /**@{*/
+    /** \brief The byte count of current tile in the bitstream buffer,
+     */
+    uint32_t                bit_stream_data_size;
+    /**
+     * offset to the first byte of the data buffer.
+     */
+    uint32_t                bit_stream_data_offset;
+    /**
+     * see VA_SLICE_DATA_FLAG_XXX definitions
+     */
+    uint32_t                bit_stream_data_flag;
+
+    uint16_t                tile_row;
+    uint16_t                tile_column;
+    uint16_t                start_tile_idx;

```

Comment:
whey we need start_tile_idx, it's possible calc if from tile_fow tile_column?

Reply:
It is the index of the first tile in the tile group of which the current tile belongs to. The index is in raster scan order of all tiles in frame. For the first tile group, this value should be 0. This parameter can be used to uniquely identify the tile group.

Reply:
could you explain more about this? Is equals to tg_start and tg_end?
If yes, why we need tile_row and tile_column?

Reply:
they are tg_start and tg_end, and will use these names.

Reply:
tile_row/tile_column is tile row/column index, which is frame level , we need it since AV1 is tile based decoding, we use tile index info to verify which the current tile it is.